janus_server/src/aggregator.rs

```diff
@@ -2149,16 +2149,35 @@ fn aggregator_filter<C: Clock>(
         aggregate_share_routing,
         aggregate_share_responding,
         warp::cors().build(),
-        response_time_recorder,
+        response_time_recorder.clone(),
         "aggregate_share",
     );
 
+    let health_check_routing = warp::path("_health_check");
+    let health_check_responding = warp::get().and(with_cloned_value(aggregator)).then(
+        |aggregator: Arc<Aggregator<C>>| async move {
+            aggregator.datastore.check_connection_pool().await?;
+            http::Response::builder()
+                .status(StatusCode::OK)
+                .body(Vec::new())
```

Comment:
nit: [`http::Response::Builder::body`](https://docs.rs/http/latest/http/response/struct.Builder.html#method.body) lets you pass `()` so I think we can save an allocation:
```suggestion
                .body(())
```

Reply:
There's no `impl From<()> for hyper::body::Body`, but I can use a static slice for the same win.

Reply:
Note that `Vec::new()` doesn't allocate an array on the heap since it has zero length; unless the `Vec` value itself is moved to the heap, I think there is no allocation, and if that's the case then I think a static slice would also end up moved to the heap.